### DISCUSSION TITLE
Which versions of Android SDK, NDK and LLVM to use for Android deployment? (5.8.0)

### DISCUSSION THREAD

**Discussion by retsi101** [HaxeFlixel/flixel#3305] [Q&A] [answered]
I'm building a small educative apps and won't need any advanced API specifics.

Haxe (4.3.6)
Lime (8.2.1)
Flixel (5.8.0)

Thanks in advance!

**Comment by ACrazyTown** [comment, marked as answer]
There's info about building to Android on Lime's website: https://lime.openfl.org/docs/advanced-setup/android/

**Reply by retsi101** [reply]
Thank you very much for such a quick response! It's been so long since I did any Haxeflixel that I completely forgot that some issues are Lime / OpenFL related.
